changelog/v0.21.20/security-scan-slack-notification.yaml

```diff
@@ -0,0 +1,3 @@
+changelog:
+  - type: NON_USER_FACING
```

Comment:
`NEW_FEATURE`? not a big deal given that this repo is internal though, if it's my only comment i will approve anyways